FirebaseDynamicLinks/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# 10.0.0
+- [removed] Removed bare initializer from `DynamicLink`. (#10236)
```

Comment:
Should be the issue number, not the PR number.

Reply:
Fixed. Thanks!